src/machine/behavior/is/memory/ldbu.rs

```diff
@@ -1,6 +1,14 @@
 use machine::state::State;
+use machine::state::mem::ByteAt;
 
-pub fn ldbu(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn ldbu(state: &mut State, x: u8, y: u8, z: u8) {
+
+    let op1: u64 = state.gpr[y].into();
+    let op2: u64 = state.gpr[z].into();
+    let mut a = op1 + op2;
+    a = a % 2u64.pow(64);
```

Comment:
Unfortunately, this does not make any sense and will panic. If `op1 + op2` is >= 2^64, the panic will occur at the `'attempt to add with overflow'`. If the sum is < 2^64, the panic will occur in the next line, when you want compute 2^64, which is **not** in the range of `u64`, resulting in an `'attempt to multiply with overflow'`